### PR TITLE
Pulltorefresh

### PR DESCRIPTION
**Pull request by vidia**
Added the Pull to refresh functionality. 

Removed the fallback calls that used "Cary West" instead of null. I now throw an exception if that happens, which should make sure that it will never happen when we test.

### CODE REVIEW

**Comment by vidia** [comment]
Completes #46

**Comment by vidia** [comment]
In true terrible non-atomic PR fashion, I am ALSO removing the reliance on SharedPrefs for tracking the lastRoom with this.

File: app/src/main/java/xyz/jhughes/laundry/activities/MachineActivity.java [hunk 1]
```diff
@@ -152,10 +156,4 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 return super.onOptionsItemSelected(item);
         }
     }
-
-    public static String getSelected() {
```
Comment (hughesjeff) [inline app/src/main/java/xyz/jhughes/laundry/activities/MachineActivity.java:162]: So uh, this method is kinda necessary...

File: app/src/main/java/xyz/jhughes/laundry/activities/MachineActivity.java [hunk 2]
```diff
@@ -17,16 +17,17 @@
...
+/**
+ * This activity does NOT extend ScreenTracked Activity because the screen views are
+ * tracked in each fragment in the ViewPager.
+ */
 public class MachineActivity extends AppCompatActivity {
 
-    /**
-     * The current room
-     */
-    private static String currentRoom;
```
Comment (hughesjeff) [inline app/src/main/java/xyz/jhughes/laundry/activities/MachineActivity.java:30]: Should be static

**Review by hughesjeff — CHANGES_REQUESTED** [review]
This also doesn't compile because a needed method was removed.

File: app/src/main/java/xyz/jhughes/laundry/activities/MachineActivity.java (hunk 1 above)
Comment (vidia) [reply app/src/main/java/xyz/jhughes/laundry/activities/MachineActivity.java:162]: Fixed it. I didnt re-add it, but I removed the reference to it. I didnt like it being static.

**Review by hughesjeff — APPROVED** [review]

**Pull request merged**